tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.ts

```diff
@@ -152,6 +151,10 @@ class TfGraphControls extends LegacyElementMixin(PolymerElement) {
         border-spacing: 0;
       }
 
+      table tr {
+        height: 19px;
```

Comment:
Is there a border on this item making it 20px? Otherwise, 19px feels very random.

Reply:
Done.

19px was the effective height before this change.